backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -6219,4 +6219,123 @@ databaseChangeLog:
+            tableName: specimen_bodysite
+            remarks: Specific body sites or locations for specimens identified by SNOMED code and mapped to SNOMED specimen codes.
+            columns:
+              - column:
+                  name: internal_id
+                  type: uuid
+                  remarks: The internal database identifier for this entity.
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: snomed_specimen_code
+                  type: text
+                  remarks: SNOMED specimen code.
+                  constraints:
+                    unique: false
+                    nullable: false
+              - column:
+                  name: snomed_specimen_display
+                  type: text
+                  remarks: The display name of the SNOMED specimen.
+                  constraints:
+                    unique: false
+                    nullable: false
+              - column:
+                  name: snomed_site_code
+                  type: text
+                  remarks: The SNOMED code for this body site.
+                  constraints:
+                    unique: false
+                    nullable: false
+              - column:
+                  name: snomed_site_display
+                  type: text
+                  remarks: The SNOMED display for this body site.
+                  constraints:
+                    unique: false
+                    nullable: false
+              - column:
+                  name: created_at
+                  type: DATETIME
+                  remarks: The creation timestamp for this entity.
+                  constraints:
+                    nullable: false
+              - column:
+                  name: updated_at
+                  type: DATETIME
+                  remarks: The timestamp for the most recent update of this entity.
+                  constraints:
+                    nullable: false
+              - column:
+                  name: created_by
+                  type: uuid
+                  remarks: The user who created this entity.
+                  constraints:
+                    nullable: true
+                    references: api_user
+                    foreignKeyName: fk__anyentity__created_by
+              - column:
+                  name: updated_by
+                  type: uuid
+                  remarks: The user who most recently updated this entity.
+                  constraints:
+                    nullable: true
+                    references: api_user
+                    foreignKeyName: fk__anyentity__updated_by
+              - column: &soft_delete_column
+                  name: is_deleted
+                  type: boolean
+                  remarks: Flag for soft-deletion of entities (false unless the entity has been deleted).
+                  constraints:
+                    nullable: false
```

Comment:
once we've updated the object to be `AuditedEntity` instead of eternal, we can remove this.

Reply:
Will remove now.